Formula/sbom-tool.rb

```diff
@@ -0,0 +1,65 @@
+class SbomTool < Formula
+  desc "Scalable and enterprise ready tool to create SBOMs for any variety of artifacts"
+  homepage "https://github.com/microsoft/sbom-tool"
+  url "https://github.com/microsoft/sbom-tool/archive/refs/tags/v1.1.8.tar.gz"
+  sha256 "86653b76ef78ed2fb43b47fc6f3b38118f0799ebbc6d4ee92afa48ef12a997fd"
+  license "MIT"
+  head "https://github.com/microsoft/sbom-tool.git", branch: "main"
+
+  depends_on "dotnet" => :build
+  # currently does not support arm build
+  # upstream issue, https://github.com/microsoft/sbom-tool/issues/223
+  depends_on arch: :x86_64
+
+  uses_from_macos "icu4c"
+  uses_from_macos "zlib"
+
+  def install
+    bin.mkdir
+
+    ENV["DOTNET_CLI_TELEMETRY_OPTOUT"] = "true"
+
+    # the architecture is hardcoded to x64 for macOS due to to an issue with
+    # the inclusion of dynamic libraries for the self-contained executable, for
+    # details see: https://github.com/microsoft/sbom-tool/issues/223#issuecomment-1644578606
+    os = OS.mac? ? "osx" : OS.kernel_name.downcase
+    arch = if OS.mac? || Hardware::CPU.intel?
+      "x64"
+    else
+      Hardware::CPU.arch.to_s
+    end
+
+    args = %W[
+      --configuration Release
+      --output #{bin}
```

Comment:
Sorry, missed this in my earlier suggestion! This will fix the CI failure.
```suggestion
      --output #{buildpath}
```